### PR TITLE
fix-up some staticcheck error

### PR DESCRIPTION
**Pull request by tanjunchen**

### CODE REVIEW

**Comment by justinsb** [comment]
Thank you @tanjunchen 

/approve
/lgtm
/ok-to-test

File: pkg/model/openstackmodel/servergroup.go [hunk 1]
```diff
@@ -60,7 +60,7 @@ func (b *ServerGroupModelBuilder) buildInstances(c *fi.ModelBuilderContext, sg *
 	}
 	igMeta["k8s"] = b.ClusterName()
 	igMeta["KopsInstanceGroup"] = ig.Name
-	igMeta["KopsRole"] = fmt.Sprintf("%s", ig.Spec.Role)
+	igMeta["KopsRole"] = ig.Spec.Role.String()
```
Comment (justinsb) [inline pkg/model/openstackmodel/servergroup.go]: This is the line that's actually causing the error that travis is finding.  I think you can just do `string(ig.Spec.Role)`

Comment (tanjunchen) [reply pkg/model/openstackmodel/servergroup.go]: OK I will change it

**Comment by justinsb** [comment]
Thanks for making the change @tanjunchen - looks great :-)

/approve
/lgtm

**Pull request merged**